src/efi.rs

```diff
@@ -615,6 +656,29 @@ fn find_file_recursive<P: AsRef<Path>>(dir: P, target_file: &str) -> Result<Vec<
     Ok(result)
 }
 
+// Find EFI dirs under usr/lib/efi
+// for exmaple: usr/lib/efi/shim/15.8-4/EFI, usr/lib/efi/grub2/2.12-34.fc42/EFI
+fn find_all_efi_dirs(sysroot_lib: &Path) -> Result<Vec<PathBuf>> {
+    const LIBDIRS: &[&str] = &["grub2", "shim"];
```

Comment:
Can we have : `const LIBDIRS: &[&str] = &["grub2", "shim","."];
So that we can keep in things in /usr/lib/efi/<vesrion>/EFI/<files>,  EFI.json metadata can contain the rpm details.

Reply:
Maybe can remove the limitation and scan all EFI directories like `/usr/lib/efi/<name>/<version>/EFI/<files>` ?

Reply:
Let's not consider files outside of a `<name>/<version>` directory here as we need to be able to know "where" they come from / attach they to a package/source, even if it's user specified.

Reply:
> is reliance on `<name>/<version>` to remove dependency on `rpm -qf` which does not work when files are copied around?

Yes, I think it will not work. In future, will rely on that to get meta for package.